config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json

```diff
@@ -1,21 +1,76 @@
 [
   {
-    "description": "Set to 'true' if the location supports offline volume expansion.",
-    "displayname": "Offline expansion",
-    "name": "offline-expansion",
-    "default": "false",
-    "regex": "^(?:true|false)$",
-    "min-length": 4,
-    "max-length": 5,
-    "supported-values": ["true", "false"],
-    "type": "boolean",
+    "description": "Ridge CSI image (DaemonSet)",
```

Comment:
Since these parameters belong to Ridge CSI driver, I think then its not required to include `Ridge CSI` keyword in descriptions and displaynames of the parameters.

For example, `"Ridge CSI image (DaemonSet)"` -----> `DaemonSet image"`